migrations/20170816224803-generatorChangeKeywordsToTags.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+'use strict';
+module.exports = {
+  up(qi, Sequelize) {
+    /*
+     Add altering commands here.
+     Return a promise to correctly handle asynchronicity.
+
+     Example:
+     return qi.createTable('users', { id: Sequelize.INTEGER });
+    */
+    return qi.renameColumn('Generators', 'keywords', 'tags');
```

Comment:
ah, i didn't recognize the difference between generator and generatortemplate - thx